csrankings-g.csv

```diff
@@ -787,6 +787,7 @@ Guanhua Yan,Binghamton University,https://www.binghamton.edu/cs/people/ghyan.htm
 Guanjie Zheng,Shanghai Jiao Tong University,https://jhc.sjtu.edu.cn/~gjzheng/,jJpqDQIAAAAJ
 Guannan Liu,Colorado School of Mines,https://www.linkedin.com/in/guannan-liu-3583b8262/,f13rb-IAAAAJ
 Guanpeng Li,University of Iowa,http://homepage.cs.uiowa.edu/~guanpli,hIoeKKkAAAAJ
+Guanyang Wang,Rutgers University,https://guanyangwang.github.io/,fnDzqzIAAAAJ

```

Comment:
No indication of how they meet inclusion criteria.

Reply:
They are affiliated faculty with the CS department: https://theory.cs.rutgers.edu/people, and they publish in cs venues: https://dblp.org/pid/224/0093.html

Reply:
Only the affiliation matters.